beta/tests/test_sota_checkpoints.py

```diff
@@ -0,0 +1,463 @@
+import os
+import json
+import sys
+import csv
+import datetime
+from typing import Tuple, List, Optional
+
+import pytest
+import subprocess
+import re
+import shlex
+from prettytable import PrettyTable
+from collections import OrderedDict
+from yattag import Doc
+from pathlib import Path
+from beta.tests.conftest import TEST_ROOT, PROJECT_ROOT
+
+BG_COLOR_GREEN_HEX = 'ccffcc'
+BG_COLOR_YELLOW_HEX = 'ffffcc'
+BG_COLOR_RED_HEX = 'ffcccc'
+
+DIFF_TARGET_MIN_GLOBAL = -0.1
+DIFF_TARGET_MAX_GLOBAL = 0.1
+DIFF_FP32_MIN_GLOBAL = -1.0
+DIFF_FP32_MAX_GLOBAL = 0.1
+
+MODE = "TF2"
+
+EVAL_SCRIPT_NAME_MAP = {
+    "classification": "main.py",
+    "object_detection": "main.py",
+    "segmentation": "evaluation.py",
+}
+
+PRETRAINED_PARAM_AVAILABILITY = {
+    "classification": True,
+    "object_detection": False,
+    "segmentation": False,
+}
+
+DATASET_TYPE_AVAILABILITY = {
+    "classification": True,
+    "object_detection": True,
+    "segmentation": False,
+}
+
+'''
+OPENVINO_DIR = PROJECT_ROOT.parent / 'intel' / 'openvino'
```

Comment:
Do you really need OpenVINO to validate checkpoints?

Reply:
Not really, it's part of pytorch sota_test_checkpoints paths. I thought it'll help in future. I'll remove it